gdmate/visualization/pyvista_vis.py

```diff
@@ -10,14 +10,27 @@ def pv_plot_2d(mesh,field,bounds=None,ax=None,colorbar=False,**kwargs):
     Plot 2D mesh using Pyvista on a Matplotlib axes.
 
     Parameters:
-        mesh : Pyvista mesh to plot
-        field : Field to use for color.
-        bounds : List of bounds (km) by which to clip the plot (default: None)
-        ax: Matplotlib axes on which to plot the mesh (default: None)
-        colorbar: Boolean for whether to include colorbar (default: False)
+        mesh : Pyvista mesh object
+            A pyvista mesh object that contains geometrical representations
+            of surface or volume data. The mesh may also have attributes,
+            such as data values assigned to points, cells, or fields assigning
+            various information to the mesh.
+        field : float
+            The name of the field contained within the Pyvista mesh object
+            to plot. 
+        bounds : list of floats or integers
+            A list of four values that define the bounds by which to clip the 
+            plot. Successively, the list of values define the minimum x,
+            maximum x, minimum y, and maximum y bounds. (default: None)
+            List of bounds (km) by which to clip the plot (default: None)
+        ax : Matplotlib axes object 
+            Matplotlib axis on which to plot the mesh (default: None)
+        colorbar : Boolean 
```

Comment:
```suggestion
        colorbar : bool
```
I'm not sure if we should do this or not - raises the question of whether the type in the docstring should be the type as expressed in code (e.g., bool or int) or the type as written out (e.g., boolean or integer).